src/test/scala/pirate/ReadSpec.scala

```diff
@@ -104,9 +104,6 @@ object ReadSpec extends Properties {
   Read.of[java.net.URI]
   Read.of[java.net.URL]
   Read.of[Option[String]]
-
-  import Read.auto._
```

Comment:
Looks like this broke 2.10? Why the change?

Reply:
`Read.auto` does not exist in Shapeless `2.1.0`. Sorry I should have run `sbt +test` locally. I'll fix it.

Reply:
Yeah if I just do this for `Read.of[(Int, Int)]`, there's no error for `Read.of[(Int, Int)]`.

```scala
Read.of[(Int, Int)](tryRead(_ => (1, 1), "(Int, Int)"))
```